Alexander96779/Broadcaster-backend
# add user profile settings

#### What does this PR do?
-  This PR allows a user to update or to complete his or her profile.
#### Description of Task to be completed?
- Use Cloudinary to manage profile images and other media.
- Create profile middleware and profile schema.
- Create a User profile controller and implement `get user profile function` and `user update profile` function.
- Create routes and write tests.
#### How should this be manually tested?
- Clone this repo to your local PC, and go to the `ft-profile-settings` branch.
- Run `npm install` to install all the dependencies.
- Run `npm run dev` in the terminal to start the app.
- Then go to `http://localhost:5000/api/profile` using `GET` to see the profile and got to `http://localhost:5000/api/profile` using `PATCH` to update the profile.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- N/A
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/52447234/83804587-aae6e380-a6ae-11ea-9059-deae0845fa6d.png)
![image](https://user-images.githubusercontent.com/52447234/83804675-c6ea8500-a6ae-11ea-9109-f2eebc29cecd.png)
